src/test/java/io/anserini/collection/C4NoCleanCollectionWithDocNoTest.java

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
`WithDocNo` should probably be `WithDocno`?

Reply:
Yes, done